gef-ui/templates/includes/exporters-address/seperate-correspondence.njk

```diff
@@ -0,0 +1,127 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-half">
+    <form
+      method="post"
+      data-cy="form">
+      {{ govukRadios({
+        classes: "govuk-radios--inline",
+        idPrefix: "correspondence",
+        name: "correspondence",
+        fieldset: {
+          legend: {
+            text: "Is there a separate correspondence address for the exporter?",
+            classes: "govuk-fieldset__legend--s"
+          }
+        },
+        errorMessage: errors and errors.fieldErrors.correspondence and {
+          text: errors.fieldErrors.correspondence.text,
+          attributes: {
+            'data-cy': 'correspondence-error'
+          }
+        },
+        items: [
+          {
+            value: true,
+            text: "Yes",
+            checked: correspondence === "true",
+            attributes: {
+              'data-cy': 'correspondence-yes',
+              onClick: "correspondenceAddress.toggle(false)"
+            }
+          },
+          {
+            value: false,
+            text: "No",
+            checked: correspondence === "false",
+            attributes: {
+              'data-cy': 'correspondence-no',
+              onClick: "correspondenceAddress.toggle(true)"
```

Comment:
client side JS isn't good for accessibility - i'm not sure on the GEF design here, but in Portal we used a pattern where depending on what radio button you click, a different NJK file can be rendered - like a show/hide. Avoids having client side js, better for accessibility: https://design-system.service.gov.uk/components/radios/conditional-reveal/index.html

Then each radio button can be like this with the `conditional` option

```
{
  value: "test",
  text: "Test",
  conditional: {
    html: someOtherFormFields.render(user)
  }
}
```

Reply:
Hey Tony.. So i looked into the conditional reveal before implementing my code. I noticed that there was no way i could update it so that it would match our designs (ie. horizontal layout instead of vertical). So I ended up creating a small method to cater for this. Also, I catered for accessibility when coding it, so that all hidden inputs are revealed when JS is turned off in the browser. let me know what you think?

Reply:
OK as long as it works - in Portal, we had same scenario - did a similar approach with client side JS originally; all seemed well, but it would act differently in IE and screen readers. We ended up changing the design to use the radio reveal component. So would just be aware of that ...happy to get it merged for now

Reply:
Cheers Tony.. Will test this further later on